client/src/app/core/services/theme.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { ConfigService } from './config.service';
+
+/**
+ * Service to set the theme for the OpenSlides.
+ * Read related data from server,
+ * the server sends the value of the new theme --> the new theme has only to be added to the body.
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class ThemeService {
+    /**
+     * Here it will subscribe to the observer from config-service to read data.
+     * @param configService must be injected to get the data from server.
+     */
+    public constructor(private configService: ConfigService) {
```

Comment:
visibility of parameters in the constructor is only required when you need to the parameter in other methods than the constructor itself.
remove `private`